etstool.py

```diff
@@ -271,6 +284,45 @@ def update(runtime, toolkit, environment):
     click.echo('Done update')
 
 
+@cli.command()
+@click.option('--runtime', default='3.6', help="Python version to use")
+@click.option('--toolkit', default='pyqt', help="Toolkit and API to use")
+@click.option('--environment', default=None, help="EDM environment to use")
+def api_docs(runtime, toolkit, environment):
+    parameters = get_parameters(runtime, toolkit, environment)
+    packages = ' '.join(doc_dependencies)
+    ignore = ' '.join(doc_ignore)
+    commands = [
+        "edm install -y -e {environment} " + packages,
+        "edm run -e {environment} -- pip install -r doc-src-requirements.txt --no-dependencies",
+    ]
+    click.echo("Installing documentation tools in  '{environment}'".format(
+        **parameters))
+    execute(commands, parameters)
+    click.echo('Done installing documentation tools')
+
+    click.echo("Regenerating API docs in  '{environment}'".format(**parameters))
+    if os.path.exists('docs/source/api/'):
+        rmtree('docs/source/api/')
+    os.makedirs('docs/source/api/')
```

Comment:
You should probably use `os.path.join` for these paths.

Reply:
Fixed.